### PR TITLE
README + Config Update

### PR DESCRIPTION
**Pull request by HuckMatt**

### CODE REVIEW

**Comment by HuckMatt** [comment]
@eric-giannini please review

File: README.md [hunk 1]
```diff
@@ -1,36 +1,68 @@
...
 ![pic](/Capture.PNG)
-Feel free to fork any of the projects here to use on your own and if you have any code improvements, make a pull request and the request will be reviewed by one of our admins.
 
-For a sample reference implentation using Symbl, take a look at our [Platform](https://platform.symbl.ai).
+## Conclusion
+
+## Community
+
+If you liked our integration guide, please star our repo! If you have any questions, feel free to reach out to us at devrelations@symbl.ai or through our Community Slack at https://developer.symbl.ai/community/slack or our [developer community][developer_community]. 
+This library is released under the [MIT License][license]
```
Comment (eric-giannini) [inline README.md:108]: Please reformat. See below.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,113 @@
...
+
+Symbl's APIs empower developers to enable: 
+- **Real-time** analysis of free-flowing discussions to automatically surface highly relevant summary discussion topics, contextual insights, suggestive action items, follow-ups, decisions, and questions.\
+- **Voice APIs** that makes it easy to add AI-powered conversational intelligence to either [telephony][telephony] or [WebSocket][websocket] interfaces.
+- **Conversation APIs** that provide a REST interface for managing and processing your conversation data.
+- **Summary UI** with a fully customizable and editable reference experience that indexes a searchable transcript and shows generated actionable insights, topics, timecodes, and speaker information.
+<hr />
+Enable Symbl for Twilio Flex Calls
+<hr />
+ * [Introduction](#introduction)
```
Comment (eric-giannini) [inline README.md:19]: Please remove: 

```
[Pre-requisites](#pre-requisites)
  * [Setup and Deploy](#setupanddeploy)
  * [Function Parameters](#functionparameters)
  * [Dependencies](#dependencies)
  * [References](#references)
  ```

File: README.md [hunk 3]
```diff
@@ -1,36 +1,113 @@
...
+ * [Dependencies](#dependencies)
+ * [References](#references)
+ * [Conclusion](#conclusion)
+ * [Community](#community)
+
+## Introduction
+
+This is a sample implementation of Symbl's Websocket API using Twilio Flex as an inbound adapter for streaming audio.  These instructions use Twilio Media Streams configured in the Twilio Studio and ngrok to expose a local endpoint.  
+
+## Pre-requisites
```
Comment (eric-giannini) [inline README.md:32]: These should be subsections of ##Integration

**Comment by eric-giannini** [comment]
@HuckMatt #lgtm Let's merge. I really like the covenant / badge for WebSockets!

File: README.md [hunk 4]
```diff
@@ -1,36 +1,68 @@
...
+## Conclusion
+
+## Community
+
+If you liked our integration guide, please star our repo! If you have any questions, feel free to reach out to us at devrelations@symbl.ai or through our Community Slack at https://developer.symbl.ai/community/slack or our [developer community][developer_community]. 
+This library is released under the [MIT License][license]
+[license]: LICENSE.txt
+[websocket]: https://docs.symbl.ai/docs/streamingapi/overview/introduction
+[developer_community]: https://community.symbl.ai/?_ga=2.134156042.526040298.1609788827-1505817196.1609788827
+[signup]: https://platform.symbl.ai/?_ga=2.63499307.526040298.1609788827-1505817196.1609788827
```
Comment (eric-giannini) [inline README.md:115]: Great!

**Review by eric-giannini — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (eric-giannini) [reply README.md:115]: May you add Slack please? `[slack]: https://join.slack.com/t/symbldotai/shared_invite/zt-4sic2s11-D3x496pll8UHSJ89cm78CA`

**Pull request merged**